test/Parser/testnetcdfParse.m

```diff
@@ -15,8 +15,8 @@ function test_no_error_reading_anmn(~, anmn_file)
             assert_burst_sampling(data);
         end
 
-        function test_no_error_reading_abos(~, abos_file)
-            data = netcdfParse({abos_file}, '');
+        function test_no_error_reading_dwm(~, dwm_file)
```

Comment:
you will need to update this everywhere this function is called too.

Reply:
This function is not called "directly" by its name. It is called as part of the unittest suite. It's not "referenced anywhere else